std_msgs/Header header

float64[] linearVelocityLeft
float64[] linearVelocityRight
float64[] angularVelocityLeft
float64[] angularVelocityRight

float64 gripperLeft
float64 gripperRight